### ISSUE TITLE
Issue include view on shortcode

### ISSUE THREAD

**Issue by mrcrandell** [wpbones/WPBones#8]
Is it possible to include a view (along with passing variables to it), attach it to a variable, but not render it. I would like to then attach the view variable to an email.

Example:
`$formDataEmail = HabitatOaklandPlugin()->view( 'emails.service-request-form' )->with( [ 'form_data' => $form_data, 'has_error' => $has_error ] );
$message = $mgClient->sendMessage($domain, array(
  'from'    => '',
  'to' => '',
  'subject' => Subject',
  'html'    => $formDataEmail
));
`
Right now, it will show the view on the page which I don't want to happen. Thank you.

**Comment by gfazioli** [comment]
@mrcrandell Hi there, I'm working on that. It will be released on `0.9.3` (probably today or tomorrow). Anyway, in the meanwhile, you can use an easy workaround:

```php
ob_start();
HabitatOaklandPlugin()
                     ->view( 'emails.service-request-form' )
                     ->with( [ 'form_data' => $form_data, 'has_error' => $has_error ] )
                     ->render(); 
$formDataEmail = ob_get_contents();
ob_end_clean();

$message = $mgClient->sendMessage($domain, array( 'from' => '', 'to' => '', 'subject' => Subject', 'html' => $formDataEmail ));
```

In the next release, you will be able to use:

```php
$formDataEmail = HabitatOaklandPlugin()
                             ->view( 'emails.service-request-form' )
                             ->with( [ 'form_data' => $form_data, 'has_error' => $has_error ] )
                             ->render( true ); 

// or...

$formDataEmail = HabitatOaklandPlugin()
                             ->view( 'emails.service-request-form' )
                             ->with( [ 'form_data' => $form_data, 'has_error' => $has_error ] )
                             ->toHTML(); 

```

Let me know if everything works fine

**Comment by mrcrandell** [comment]
Awesome! I'll test everything out. Thank you for the fast response. Great work!
